plugin/schema.json

```diff
@@ -0,0 +1,1234 @@
+{
```

Comment:
Note this file isn't used, it's just a convenience for testing with vs.code, useful to have in the history of this PR so changes can be tracked as it evolves.

It should be removed before merge